protos/remote_id/remote_id.proto

```diff
@@ -0,0 +1,125 @@
+    /*
+     * Update the Location structure sent with the location packet
+     */
+    rpc SetLocation(SetLocationRequest) returns(SetLocationResponse) { option (mavsdk.options.async_type) = SYNC; }
+    /*
+     * Update the System structure sent with the system packet
+     */
+    rpc SetSystem(SetSystemRequest) returns(SetSystemResponse) { option (mavsdk.options.async_type) = SYNC; }
+    /*
+     * Update the OperatorId structure sent with the operator_id packet
+     */
+    rpc SetOperatorId(SetOperatorIdRequest) returns(SetOperatorIdResponse) { option (mavsdk.options.async_type) = SYNC; }
+    /*
+     * Update the SetSelfId structure sent with the self_id packet
+     */
+    rpc SetSelfId(SetSelfIdRequest) returns(SetSelfIdResponse) { option (mavsdk.options.async_type) = SYNC; }
+}
+
+message SetBasicIdRequest {
+    BasicId basic_id = 1; // Desired basic_id
+}
+message SetBasicIdResponse {
+    RemoteIdResult remote_id_result = 1;
+}
+
+message SetLocationRequest {
+    Location location = 1; // Desidered location
+}
+message SetLocationResponse {
+    RemoteIdResult remote_id_result = 1;
+}
+
+message SetSystemRequest {
+    SystemId system = 1; // Desidered system
+}
+message SetSystemResponse {
+    RemoteIdResult remote_id_result = 1;
+}
+
+message SetOperatorIdRequest {
+    OperatorId system = 1; // Desidered operator_id
+}
+message SetOperatorIdResponse {
+    RemoteIdResult remote_id_result = 1;
+}
+
+message SetSelfIdRequest {
+    SelfId self_id = 1; // Desidered self_id
+}
+message SetSelfIdResponse {
+    RemoteIdResult remote_id_result = 1;
+}
+
+message BasicId {
+    uint32 id_type = 1; // Indicates the format for the uas_id field of this message.
+    uint32 ua_type = 2; // Indicates the type of UA (Unmanned Aircraft).
+    string uas_id = 3; // UAS ID following the format specified by id_type.
+}
+
+message Location {
+    uint32 status = 1; // Indicates whether the unmanned aircraft is on the ground or in the air.
+    uint32 direction_deg = 2; // Direction over ground measured clockwise from true North: 0 - 359 deg.
+    uint32 speed_horizontal_cm_s = 3; // Ground speed. Positive only. cm/s
+    int32 speed_vertical_cm_s = 4; // The vertical speed. Up is positive. cm/s
+    float latitude_deg = 5; // Current latitude of the unmanned aircraft
+    float longitude_deg = 6; // Current longitude of the unmanned aircraft
+    float altitude_barometric_m = 7; // The altitude calculated from the barometric pressure.
+    float altitude_geodetic_m = 8; // The geodetic altitude as defined by WGS84.
+    uint32 height_reference = 9; // Indicates the reference point for the height field.
+    float height_m = 10; // The current height of the unmanned aircraft. As indicated by height_reference.
+    float timestamp_s = 11; // Seconds after the full hour with reference to UTC time.
+    uint32 timestamp_accuracy = 12; // The accuracy of the timestamps.
```

Comment:
Gosh, this is again so ugly in the MAVLink message. It's an enum in steps of 0.1s to signal accuracy. I wonder who thought this would be a good idea...

In any case, my suggestion would be to make the API human usable by just doing `timestamp_accuracy_s`, so in seconds, and then do the ugly conversion inside MAVSDK.

Reply:
i think the the suggested changes are completed.